docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+services:
```

Comment:
I'm curious why docker compose is needed if the only service being run is a single container.

Reply:
Using Docker compose is allowing me to work in my local git clone but use the cached dev env.
https://github.com/ArjanCodes/2022-docker

I could also have used --reload in a run command, but I like the syncing this method provides, and the configuration of the run via a file vs a long command line.